source/less/blocks/videopresentation.less

```
@@ -6,8 +6,21 @@
 
 .videopresentation {
   margin: 62px 0 120px 0;
+  position: relative;
 }
 
 .videopresentation__video {
   padding-top: 64px;
```

```suggestion
  margin-top: 64px;
```
Отступ падингами почти всегда нужен когда требуется сохранить фон элемента вокруг.